tests/components/conversation/snapshots/test_init.ambr

```diff
@@ -563,7 +563,7 @@
       'speech': dict({
         'plain': dict({
           'extra_data': None,
-          'speech': 'Sorry, I am not aware of any device called kitchen light',
+          'speech': 'Sorry, I am not aware of any device called kitchen',
```

Comment:
For reference, which upstream PR/commit caused the need to change this snapshot?

Reply:
This is due to a change in core with how recognition results are ordered. More literal text match pushes a result higher in the list, so now the "light" at the end is not being considered as part of the name.

Reply:
Thanks